tensorflow/core/api_def/base_api/api_def_RequantizePerChannel.pbtxt

```diff
@@ -0,0 +1,65 @@
+op {
+graph_op_name: "RequantizePerChannel"
```

Comment:
There should be indentation here. Did you use the right formatter?

Reply:
No worries! I am not trusting formatters now! Its on me and sorry about that. Will be easier next time around with my PRs.